### PR TITLE
Sidebar: show latest release badge

### PR DESCRIPTION
**Pull request by simon04**

### CODE REVIEW

File: content/guides/installation.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ The next section tells you how to install webpack locally in a project.
 
 ### Local Installation
 
+The latest released webpack version is: [![GitHub release](https://img.shields.io/github/release/webpack/webpack.svg?style=flat-square)](https://github.com/webpack/webpack/releases)
```
Comment (skipjack) [inline content/guides/installation.md]: Maybe `The latest webpack release is: ...` would sound slightly better. Feel free to ignore though, somewhat nitpicky.

File: components/sidebar/sidebar.jsx [hunk 2]
```diff
@@ -35,7 +35,6 @@ export default class Sidebar extends Component {
           height={ availableHeight } />
 
         <div className="sidebar__inner">
-          <img src="https://img.shields.io/badge/webpack-v2.3.3-blue.svg" />
```
Comment (skipjack) [inline components/sidebar/sidebar.jsx]: Not sure if we should entirely remove it from here without having it somewhere else that's consistently displayed. This came up a few times a while back which is why we added it to the sidebar for lack of a better place. @bebraw @TheLarkInn what do you think?

Comment (bebraw) [inline components/sidebar/sidebar.jsx]: Leave it for now. You still want to show the version globally I think and sidebar is a nice place for that.

Comment (skipjack) [inline components/sidebar/sidebar.jsx]: @simon04 maybe just update this one to be the dynamic version you used in the installation guide? Unless we want to bump this manually as new docs are added...

Comment (simon04) [inline components/sidebar/sidebar.jsx]: Alright. I'll move the badge back to the sidebar.

Comment (skipjack) [inline components/sidebar/sidebar.jsx]: I think you could have it in both places.

Comment (simon04) [inline components/sidebar/sidebar.jsx]: I agree. The badge in the installation guide is back with 28c89cd.

**Pull request merged**